packages/form-js-viewer/src/render/components/form-fields/FilePicker.js

```diff
@@ -1,8 +1,86 @@
+import { formFieldClasses } from '../Util';
+import { Label } from '../Label';
+import { Errors } from '../Errors';
+import { useRef, useState } from 'preact/hooks';
+import { useSingleLineTemplateEvaluation } from '../../hooks';
+
+const type = 'filepicker';
+
 /**
+ * @typedef Props
+ * @property {(props: { value: string }) => void} onChange
+ * @property {string} domId
+ * @property {string[]} errors
+ * @property {boolean} disabled
+ * @property {boolean} readonly
+ * @property {boolean} required
+ * @property {Object} field
+ * @property {string} field.id
+ * @property {string} [field.label]
+ * @property {string} [field.accept]
+ * @property {boolean} [field.multiple]
+ *
+ * @param {Props} props
  * @returns {import("preact").JSX.Element}
  */
-export function FilePicker() {
-  return null;
+export function FilePicker(props) {
+  /** @type {import("preact/hooks").Ref<HTMLInputElement>} */
+  const fileInputRef = useRef(null);
+  /** @type {[File[],import("preact/hooks").StateUpdater<File[]>]} */
+  const [selectedFiles, setSelectedFiles] = useState([]);
```

Comment:
Can be handled later, but I can see that the file uploader caches the values during form import and reset. This might not matter hugely now because things are going to change once the files are properly handled, but this would clear it:

```
  eventBus.on('import.done', () => setSelectedFiles([]));
  eventBus.on('reset', () => setSelectedFiles([])));
```

Reply:
I guess it doesn't hurt to add it